expyfun/stimuli/_tracker.py

```diff
@@ -239,6 +248,22 @@ def respond(self, correct):
                 if self._direction <= 0:
                     self._direction = 1
 
+        if self._repeat_limit == 'reversals':
+            if self._x[-1] == self._x_min:
```

Comment:
... and in both cases, you'd record 7 reversals, and not want to use any of them in a subsequent calculation.

Let's take an example where a subject starts out bad, but eventually figures it out. In the current scheme, the full run could look like this (12 reversals):

```
-------------------
  /\/\/\/\
 /        \  /\
/          \/  \/\/
```
My proposal would be that it should look like this:
```
-------------------
  /‾‾‾‾‾‾\
 /        \  /\
/          \/  \/\/
```
I think in either case you should only use the last 5 reversals in a calculation (and get an error if you use the built-in threshold calculators and include these bound-limited points).

Maybe this gets more complicated when it's not 1-up, 1-down, I'm not sure...